manifests/bridges.pp

```diff
@@ -10,7 +10,7 @@
     $bridges = $interfaces.filter |$items| { $items =~ $bridgenames }
 
     $bridges.each |String $bridge| {
-      $bridge_rulename = regsubst($bridge, '-', '_', 'G')
+      $bridge_rulename = regsubst($bridge, '-|:', '_', 'G')
```

Comment:
For single char, I trend to prefer `[]` to `|` because it feels more readable:
```suggestion
      $bridge_rulename = regsubst($bridge, '[-:]', '_', 'G')
```
That being said, a `-` is tricky in `[]` and we have two chars only so not a big deal.